setup.py

```diff
@@ -152,7 +152,7 @@
             *extra_plugins['write'],
         ],
         'datacube.plugins.index': [
-            'default = datacube.index.postgres.index:index_driver_init',
+            'postgres = datacube.index.postgres.index:index_driver_init',
```

Comment:
Would we still need to have a 'default' here as well?

Reply:
No "default" and "legacy" are added as aliases.